jupyter_server_proxy/config.py

```diff
@@ -203,3 +203,51 @@ def host_whitelist(handler, host):
     @default("host_whitelist")
     def _host_whitelist_default(self):
         return ["localhost", "127.0.0.1"]
+
+    keyfile = Unicode(
+        "",
+        help="""
+        Path to optional SSL key.
+
+        Use with `https=True` and `certfile`.
+        """,
+        config=True
+    )
+
+    certfile = Unicode(
+        "",
+        help="""
+        Path to optional SSL cert.
+
+        Use with `https=True` and `keyfile`.
+        """,
+        config=True
+    )
+
+    cafile = Unicode(
+        "",
+        help="""
+        Path to optional CA file.
+
+        Use with `https=True`.
+        """,
+        config=True
+    )
+
+    https = Bool(
+        False,
+        help="""
+        Whether to use SSL for forwarded client requests.
+
+        If this is set to `True` then you should provide a path to an SSL key,
+        cert, and CA.  Use this if the proxied service expects to service
+        requests over SSL.
+        """,
+        config=True
+    )
```

Comment:
Is it possible to get all these SSL informations from mybinder (which uses SSL)?